src/testHelper.js

```diff
@@ -62,9 +63,20 @@ export async function createIntegrationTestUser() {
  * @param {Object} user created in #createIntegrationTestUser()
  * @returns {Promise}
  */
+
 export async function removeIntegrationTestUser(user) {
   return removeTestUser({
     ...user,
     conversationServiceUrl: URL_CONVO_SERVICE_INTEGRATION,
   });
 }
+
+/**
+ * Constructs a Hydra ID for a given UUID.
+ *
+ * @param {String} UUID Person UUID
+ * @returns {string}
+ */
+export function getPersonHydraID(UUID) {
```

Comment:
first, I tried to add the `Hydra` ID to the object. However deserializing the user during removing the user threw some errors, since that property does not exist to begin with. Therefore, I decided to create a common method to use. I'm sure this method will be utilized for other suites as we implement more test cases.